CMakeLists.txt

```
@@ -16,6 +16,47 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(rosidl_default_generators REQUIRED)
 
+set(protobuf_MODULE_COMPATIBLE TRUE)
+find_package(Protobuf CONFIG REQUIRED)
+message(STATUS "Using protobuf ${Protobuf_VERSION}")
+
+set(_PROTOBUF_LIBPROTOBUF protobuf::libprotobuf)
+set(_REFLECTION gRPC::grpc++_reflection)
+set(_PROTOBUF_PROTOC $<TARGET_FILE:protobuf::protoc>)
+
+find_package(gRPC CONFIG REQUIRED)
+message(STATUS "Using gRPC ${gRPC_VERSION}")
+
+set(_GRPC_GRPCPP gRPC::grpc++)
+set(_GRPC_CPP_PLUGIN_EXECUTABLE $<TARGET_FILE:gRPC::grpc_cpp_plugin>)
+
+# Proto file
+get_filename_component(aruku_proto "./proto/aruku.proto" ABSOLUTE)
+get_filename_component(aruku_proto_path "${aruku_proto}" PATH)
```

Please use `DIRECTORY`, see [this](https://cmake.org/cmake/help/latest/command/get_filename_component.html).
```suggestion
get_filename_component(aruku_proto_path "${aruku_proto}" DIRECTORY)
```